### ISSUE TITLE
UglifyJS2 error: Name expected

### ISSUE THREAD

**Issue by pushred** [pushred/viewport-focus#1]
When compressing with UglifyJS2 2.8.29 in a Webpack build, this error results:

```
ERROR in vendor.js from UglifyJs
Name expected [vendor.js:42298,8]
```

The impacted line:

```js
const { focalPoint = 'middle', offset } = options;
```

**Comment by pushred** [comment]
It's a transpilation issue, UglifyJS2 doesn't support ES6 syntax. I provide a transpiled version in `dist` but the version I provide on the `module` field is not. It's possible to use that version, but it requires tweaking the typical babel-loader configuration to transpile it prior to uglifying, e.g.

```js
module: {
  rules: [{
    test: /\.(js|jsx)$/,
    exclude: /node_modules\/(?!map-obj|vestwell).*/,
  }]
...
```

I do include a pre-transpiled version, specified as the `main` field in the package config. Webpack prioritizes `module` however, which is not transpiled. I'll add a `browser` field [to override that](https://webpack.js.org/configuration/resolve/#resolve-aliasfields).

[uglify-es adds ES6 support](https://github.com/mishoo/UglifyJS2/tree/harmony) so this will partly be solved soon. But to help ensure the build is browser-friendly, I'm also adding babel-preset-env config to target:

```
"browsers": ["last 2 versions"]
```

**Comment by pushred** [comment]
fixed by 989b9fc08a3e1bdf141d0f13a2fc44645b6dbdac
